poc/idpf.sage

```diff
@@ -117,6 +119,32 @@ def test_idpf(Idpf, alpha, level, prefixes):
                 alpha, prefix, level, got, want))
 
 
+def gen_test_vec(Idpf, alpha, test_vec_instance):
+    beta_inner = [[Idpf.FieldInner(1)] * Idpf.VALUE_LEN] * (Idpf.BITS-1)
+    beta_leaf = [Idpf.FieldLeaf(1)] * Idpf.VALUE_LEN
+    (public_share, keys) = Idpf.gen(alpha, beta_inner, beta_leaf)
+
+    printable_beta_inner = [
+        [ str(elem.as_unsigned()) for elem in value ] for value in beta_inner
+    ]
+    printable_beta_leaf = [ str(elem.as_unsigned()) for elem in beta_leaf ]
+    printable_keys = [ key.hex() for key in keys ]
+    test_vec = {
+        'bits': int(Idpf.BITS),
+        'beta_inner': printable_beta_inner,
+        'beta_leaf': printable_beta_leaf,
+        'public_share': public_share.hex(),
+        'keys': printable_keys,
+    }
+
+    os.system('mkdir -p test_vec/{}'.format(DRAFT))
+    with open('test_vec/{}/{}_{}.json'.format(
+        DRAFT, Idpf.__name__, test_vec_instance), 'w') as f:
```

Comment:
We can punt on this for now, but we'll soon need to include the PRG name here.

Reply:
Done.